chirp/wxui/bankedit.py

```diff
@@ -80,7 +80,8 @@ def __init__(self, radio, *a, **k):
         self._grid.CreateGrid(
             self._features.memory_bounds[1] - self._features.memory_bounds[0] +
             1, len(self._col_defs))
-        self._grid.SetSelectionMode(wx.grid.Grid.GridSelectNone)
+        # GridSelectNone only available in >=4.2.0, which is not in requirements.txt for Linux
+        #self._grid.SetSelectionMode(wx.grid.Grid.GridSelectNone)
```

Comment:
This breaks it for everyone else. Can you please just guard it this?
```
if hasattr(wx.grid.Grid.GridSelectNone):
    self._grid.SetSelectionMode(wx.grid.Grid.GridSelectNone)
```

Reply:
So done.

I'm sure there are multiple reasons why 4.2.0 isn't used for Linux, but not clear what they would be - just distro standard?

Reply:
Yeah, all the current debian-based distros have 4.0.7 and installing 4.2.0 via binary wheel is broken on linux. It's...annoying, but once bookworm comes out, we'll hopefully be able to lock to 4.2.0 everywhere shortly thereafter.

Thanks for the change, I'll hit this later today.